packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -326,22 +303,20 @@ export default class GoogleCalendarService implements Calendar {
               },
             },
             (err, apires) => {
-              if (err) {
+              if (err || !apires?.data.calendars) {
```

Comment:
`!apires?.data.calendars` doesn't mean `err` will be defined, would it be better to have another check?

```
if (!apires?.data.calendars) {
	reject("No calendar data found...")
	return;
}

Reply:
That check passes if any of those is true

Reply:
Here are the instructions to [setup Google](https://github.com/calcom/cal.com#obtaining-the-google-api-credentials)

Reply:
Yes but if `err` is `null` and it hits on `!apires?.data.calendars` then you'll be rejecting on null (`reject(null)`) and there will be no context in why the error occurred.

Reply:
Agreed. will update